tcf_website/templates/search/searchbar.html

```diff
@@ -1,13 +1,536 @@
-<!-- tCF search -->
-<!-- To be imported into templates (navbar, landing page) -->
-<form class="form-inline flex-grow-1" action="/search" method="get">
+<!-- searchbar.html -->
+ 
+<form class="form-inline flex-grow-1" action="{% url 'search' %}" method="get">
   <div class="input-group flex-grow-1">
-    <input type="search" class="form-control border border-right-0"
-            placeholder="Search for a class or professor..." aria-label="Search" name="q" value="{{query}}">
+    <input 
+      type="search" 
+      class="form-control border border-right-0"
+      placeholder="Search for a class or professor..." 
+      aria-label="Search" 
+      name="q" 
+      value="{{ query }}"
+    >
     <div class="input-group-append">
-      <button class="btn btn-outline-secondary border border-left-0" id="search-button" aria-label="Search" type="submit">
-          <i class="fa fa-search" aria-hidden="true"></i>
+      <!-- Search Button -->
+      <button 
+        class="btn btn-outline-secondary border border-left-0" 
+        id="search-button" 
+        aria-label="Search" 
+        type="submit"
+      >
+        <i class="fa fa-search" aria-hidden="true"></i>
       </button>
     </div>
+    <div class="dropdown-wrapper">
```

Comment:
Separate out to another html to { % include ___ with ___ % } to separate functionality of dropdown and searchbar.